Automattic/newspack-popups
# Update/migration methods class

### All Submissions:






### Changes proposed in this Pull Request:



This PR

* Moves all methods related to migration to its own class
* Runs the migration from configuration to criteria in all segments at once

### How to test the changes in this Pull Request:

1. Set up a site with segments in master
2. Checkout this branch
3. Make sure migration worked

### Other information:
